src/stories/containers/finances-overview/Components/YearPicker/YearPicker.tsx

```diff
@@ -1,15 +1,75 @@
+import styled from '@emotion/styled';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { CustomButton } from '@ses/components/custom-button/custom-button';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import { ButtonType } from '@ses/core/enums/button-type.enum';
+import { ButtonPickerStyle } from '@ses/core/utils/share-style';
 import React from 'react';
+import lightTheme from 'styles/theme/light';
 
-const YearPicker = () => {
-  const years = [2021, 2022, 2023];
+interface Props {
+  years?: number[];
+  yearSelect: number;
+  handleOnclick: (year: number) => void;
+}
+
+const YearPicker = ({ years = [], yearSelect, handleOnclick }: Props) => {
+  const isUpMobile = useMediaQuery(lightTheme.breakpoints.up('table_834'));
+  const { isLight } = useThemeContext();
+  const onclick = (year: number) => () => {
+    handleOnclick(year);
+  };
 
   return (
-    <div style={{ display: 'flex' }}>
-      {years.map((year) => (
-        <div key={year}>{year}</div>
+    <Container
+      style={{
+        display: 'flex',
+        marginBottom: 40,
+      }}
+    >
+      {years?.map((year) => (
+        <ContainerButtons key={year}>
+          <CustomButton
+            onClick={onclick(year)}
+            buttonType={ButtonType.Secondary}
+            widthText="100%"
+            label={year.toString()}
+            style={{
+              background: ButtonPickerStyle(isLight, year, yearSelect).background,
```

Comment:
Can this be stored in a variable so you don't need to call this function everytime

Reply:
No sure if this will make all the difference, we can talk about it